[docs] Batch 20 - Aggregation functions, part 1 (#15)
edgar2020/druid: docs/querying/sql-functions.md

> * **Syntax**: `APPROX_COUNT_DISTINCT(expr)`

Reviewer: for line 140, worth it to include "using an approximate algorithm."?

* **Syntax**: `APPROX_COUNT_DISTINCT_BUILTIN(expr)`
* **Function type:** Aggregation

> <details><summary>Example</summary>

Reviewer: for line 166, should we recommend `APPROX_COUNT_DISTINCT_DS_HLL`?

* **Syntax**: `MAX(expr)`
* **Function type:** Aggregation

<details><summary>Example</summary>

> The following example calculates the maximum minutes of delay for a particular airlines in `flight-carriers`:

Reviewer: ```suggestion
The following example calculates the maximum delay in minutes for an airline in `flight-carriers`:
```

* **Syntax**: `MIN(expr)`
* **Function type:** Aggregation

<details><summary>Example</summary>

> The following example calculates the minimum minutes of delay for a particular airlines in `flight-carriers`:

Reviewer: ```suggestion
The following example calculates the minimum delay in minutes for an airline in `flight-carriers`:
```

## STDDEV_POP

Calculates the population standard deviation of a set of values.  
Requires the [`druid-stats` extension](../development/extensions-core/stats.md).

* **Syntax**: `STDDEV_POP(expr)`
* **Function type:** Aggregation

<details><summary>Example</summary>

> The following example calculates the population standard deviation for minutes of delay by a particular airlines in `flight-carriers`:

Reviewer: ```suggestion
The following example calculates the population standard deviation for minutes of delay for an airline in `flight-carriers`:
```

```sql
SELECT STDDEV_POP("DepDelayMinutes") AS sd_delay
FROM "flight-carriers"
WHERE "Reporting_Airline" = 'AA'
```

Returns the following:

| `sd_delay` |
| -- |
| `27.083557` |

</details>

[Learn more](sql-aggregations.md)

## STDDEV_SAMP

Calculates the sample standard deviation of a set of values.  
Requires the [`druid-stats` extension](../development/extensions-core/stats.md).

* **Syntax**: `STDDEV_SAMP(expr)`
* **Function type:** Aggregation

<details><summary>Example</summary>

> The following example calculates the sample standard deviation for minutes of delay by a particular airlines in `flight-carriers`:

Reviewer: ```suggestion
The following example calculates the sample standard deviation for minutes of delay for an airline in `flight-carriers`:
```

* **Syntax**: `SUM(expr)`
* **Function type:** Aggregation

<details><summary>Example</summary>

> The following example calculates the total minutes of delay for a particular airlines in `flight-carriers`:

Reviewer: ```suggestion
The following example calculates the total minutes of delay for an airline in `flight-carriers`:
```

## VAR_POP

Calculates the population variance of a set of values.  
Requires the [`druid-stats` extension](../development/extensions-core/stats.md).

* **Syntax**: `VAR_POP(expr)`
* **Function type:** Aggregation

<details><summary>Example</summary>

The following example calculates the population variance for minutes of delay by a particular airlines in `flight-carriers`:

```sql
SELECT VAR_POP("DepDelayMinutes") AS varpop_delay
FROM "flight-carriers"
WHERE "Reporting_Airline" = 'AA'
```

Returns the following:

| `varpop_delay` |
| -- |
| `733.51908` |

</details>

[Learn more](sql-aggregations.md)

## VAR_SAMP

Calculates the sample variance of a set of values.  
Requires the [`druid-stats` extension](../development/extensions-core/stats.md).

* **Syntax**: `VAR_SAMP(expr)`
* **Function type:** Aggregation

<details><summary>Example</summary>

> The following example calculates the sample variance for minutes of delay by a particular airlines in `flight-carriers`:

Reviewer: ```suggestion
The following example calculates the sample variance for minutes of delay for an airline in `flight-carriers`:
```
